### ISSUE TITLE
No learning

### ISSUE THREAD

**Issue by parajain** [srviest/char-cnn-pytorch#5]
I am trying the code, but precision is not improving. Model keeps on selecting only one of the classes:

![image](https://user-images.githubusercontent.com/8446082/38074773-9277fda6-334d-11e8-976d-5706d0e2547e.png)

![image](https://user-images.githubusercontent.com/8446082/38074769-88830aac-334d-11e8-9a1a-df5372deac41.png)

Thanks

**Comment by srviest** [comment]
Please check #3

**Comment by parajain** [comment]
Thanks, will try smaller learning rate.
If possible can you please set the default parameters such that python train.py produces some better results? That will be helpful.
Thanks for the code :)
